Cloud_Providers.md

```diff
@@ -30,21 +31,22 @@ In order to use the SAS Viya monitoring and logging components in an Amazon EKS
 
 No other customization is needed to deploy and run the SAS Viya logging and monitoring components on Amazon EKS.
 
-### Using Amazon EKS Logging and Monitoring Applications With SAS Viya
+### Alternative Monitoring Approach: Using Amazon EKS Logging and Monitoring Applications With SAS Viya
+
+As an alternative to using the SAS Viya monitoring and logging applications, you can use CloudWatch, which is Amazon EKS's monitoring application. 
 
-If you deploy SAS Viya in an Amazon EKS environment, you can view metric information by using either the SAS Viya monitoring components or Amazon's monitoring application, CloudWatch. In order to use CloudWatch to monitor SAS Viya metrics, you must deploy the CloudWatch agent for Prometheus, make configuration changes to convert the Prometheus metrics collected from SAS Viya into performance log events, and map the log events to CloudWatch metrics. Follow the steps in the [Amazon CloudWatch Integration sample](/samples/cloudwatch/README.md) to use CloudWatch to monitor a selected group of SAS Viya metrics. Not all SAS Viya metrics are supported by this sample, because supporting all SAS Viya metrics would require every possible combination of labels from every SAS Viya source and every metric to be mapped to CloudWatch metric types. This sample might be updated in later releases to include additional combinations of dimensions for key metrics.
+If you choose to use CloudWatch to monitor SAS Viya metrics, you must deploy the CloudWatch agent for Prometheus, make configuration changes to convert the Prometheus metrics collected from SAS Viya into performance log events, and map the log events to CloudWatch metrics. Follow the steps in the [Amazon CloudWatch Integration sample](/samples/cloudwatch/README.md) to use CloudWatch to monitor a selected group of SAS Viya metrics. Not all SAS Viya metrics are supported by this sample, because supporting all SAS Viya metrics would require every possible combination of labels from every SAS Viya source and every metric to be mapped to CloudWatch metric types. This sample might be updated in later releases to include additional combinations of dimensions for key metrics.
 
-Amazon CloudWatch can also display log messages collected from Fluent Bit, which is the application used by the SAS Viya logging solution. However, current samples do not provide the configuration needed to integrate SAS Viya log processing with CloudWatch.  
+Amazon CloudWatch can also display log messages collected from Fluent Bit, which is the application used by the SAS Viya logging solution. However, there are differences in the formatting of log messages produced by SAS Viya applications and services. Although these messages are processed by the SAS Viya logging applications so that Kibana displays them using a consistent format, these messages are displayed in different formats on CloudWatch. 
 
```

Comment:
Hmmm, still not right.  Maybe:
_Amazon CloudWatch can also allow you to monitor the log messages produced by SAS Viya applications and services.   However, unlike in the Viya log monitoring solution, CloudWatch will not present all of the messages produced by SAS Viya applications and services in a single consistent format.  Some components of SAS Viya, especially several of the third-party components, do not produce log messages in the same format.  While the SAS Viya log monitoring solution standardizes these messages into the same format as the rest of the SAS Viya messages, CloudWatch does not do that. This may make it more difficult to work with the log messages when presented in CloudWatch._

Reply:
Paragraph changed to the one provided, with some minor wording tweaks